### DISCUSSION TITLE
Multiple select

### DISCUSSION THREAD

**Discussion by andrezanna** [andrewdwallo/filament-selectify#14] [Feature Requests]
I would like to have multiselect like the default Filament Select, but at the moment is not supported.
Since i'm registering trough a relation this would be really helpful.

now I use this
Select::make('categories')->multiple()->relationship(titleAttribute: 'name')->preload(),

this should become
ButtonGroup::make('categories')->options(Category::all()->pluck('name','id'))->onColor('primary')->offColor('gray')->multiple()

**Comment by andrewdwallo** [comment]
Filament has this functionality for their new `ToggleButton` component as shown [here](https://filamentphp.com/docs/3.x/forms/fields/toggle-buttons#selecting-multiple-buttons). Would this suffice for what you desire? Or are you wanting specifically for this package to provide this functionality?

**Reply by andrewdwallo** [reply]
I just wasn't aware that people would still want to use this package given that Filament has implemented similar components in the core, but I am willing to implement this feature if you desire.
